src/pages/library/dataBrowser-utils.js

```diff
@@ -80,7 +80,7 @@ const normalizeDataset = dataset => {
     dataReleasePolicy,
     contacts, curators, contributorNames,
     dataType, dataModality,
-    access: _.intersection(dataset.roles, ['reader', 'owner']).length > 0 ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
+    access: (dataset.accessLevel === 'reader' || dataset.accessLevel === 'owner') ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Comment:
FYI on another way of expressing the predicate in case it's easier to read:
```suggestion
    access: _.includes(dataset.accessLevel, ['reader', 'owner']) ? datasetAccessTypes.GRANTED : datasetAccessTypes.CONTROLLED
```

Reply:
I'd also recommend creating a `datasetAccessLevels` similarly to `datasetAccessTypes` to avoid using string literals any more than necessary.

Reply:
Thank you! I'd been trying to remember the syntax for this, this is good.

I agree on `datasetAccessLevels` and avoiding raw strings, but I might file that as tech debt since its already an issue, and my goal is to fix the UI first.